CHANGELOG.md

```diff
@@ -0,0 +1,26 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), 
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+### Added
+- Added crossword grid
+- Insert words into crossword grid
```

Comment:
"- Insert words into crossword grid"
Should be past tense:
"- Inserted words into crossword grid"

Reply:
changed tense, thanks